test/Interop/Cxx/extern-c/Inputs/extern-c.h

```diff
@@ -0,0 +1,11 @@
+@interface A
```

Comment:
Nit: technically this isn't a C++ test... but it's not a big problem.

Reply:
Oh, I guess it is. The whole point is to test extern "C". So ignore the other comment. Sorry :)

Reply:
> 

Yep, I was about to say "the extern "C" makes this c++"